gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageWriteChannel.java

```diff
@@ -20,11 +20,11 @@
 @ThreadSafe
 final class CloudStorageWriteChannel implements SeekableByteChannel {
 
-  private GcsOutputChannel channel;
+  private WriteChannel channel;
```

Comment:
Also, as for the "TODO" bellow remove or replace internal reference (also replace apiary with api client).
BTW, about the comment. do you think there is a truncate operation is available in the google cloud storage API?